orchestra/bots/tests/test_basebot.py

```diff
@@ -44,7 +44,7 @@ def test_dispatch(self):
             (r'test_command (?P<test_param>[0-9]+)', 'test_command'),
         )
 
-        def test_command(test_param):
+        def test_command(test_param, command=None):
             return text
```

Comment:
also how does this function work haha what is `text` should that be `test_param`?